tests/test_data/weightwatchers.com/weightwatcherspublic.json

```diff
@@ -29,6 +29,7 @@
     "Öl in einem Topf erhitzen, Kartoffelwürfel und Zwiebelstreifen darin ca. 5 Minuten anbraten. Paprikastücke zufügen, Tomatenmark einrühren und kurz mitbraten. Mit Brühe ablöschen und zugedeckt ca. 15 Minuten garen.",
     "Kartoffelgulasch mit Oregano, Paprikapulver, Salz und Pfeffer würzen. Würstchenscheiben und Tomatenspalten zufügen und weitere ca. 5 Minuten garen. Kartoffelgulasch mit Salz und Pfeffer abschmecken. Mit einem Klecks Schmand und Petersilie bestreut servieren."
   ],
+  "category": "WeightWatchers",
```

Comment:
This may have uncovered what I would consider to be a bug: the `category` field of this scraper returns a static string -- but the test HTML does contain (admittedly empty) `recipeCategory` in schema metadata.  I guess that means that other pages on their website do provide a `recipeCategory` value.. and so we should probably use `return self.schema.category()` as the implementation instead (or omit it to use that as the default).

Reply:
I've reported bug #1212 that explains the problem here - strictly speaking I think we should move that change into a separate PR, even though it's a small adjustment.